src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let price = 40 * days;
```

Comment:
What is 40? Let's create a variable for that value to make your code more understandable